tests/functional/empty-app/expected-stderr

```diff
@@ -0,0 +1 @@
+%AUnexpected action "run"%A
```

Comment:
Jediny datadir test, prazdna konfiguracia ... kedze pouzivame API, ... tak som kontrolu vygenerovaneho SQL presunul do unittestov - ... aby sme "zamkli" aktualny stav, ... kedze sme to rucne otestovali na produkcii.